bridge/client/app/_services/data.service.ts

```diff
@@ -153,7 +154,7 @@ export class DataService {
         map(roots => roots.map(root => Root.fromJSON(root)))
       )
       .subscribe((roots: Root[]) => {
-        service.roots = [...roots||[], ...service.roots||[]].sort((a, b) => new Date(b.time).getTime() - new Date(a.time).getTime());
+        service.roots = [...roots||[], ...service.roots||[]].sort((a, b) => DateUtil.compareTraceTimes(a, b));
```

Comment:
```suggestion
        service.roots = [...roots||[], ...service.roots||[]].sort(DateUtil.compareTraceTimes);
```

could make this even shorter when using no wrapper function